content/07.pathogenesis.md

```diff
@@ -71,6 +71,7 @@ The nsp include enzymes such as RNA-dependent RNA Polymerase (RdRp).
 The last third of the genome encodes structural proteins, including the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.
 Accessory genes can also be interspersed throughout the genome [@doi:10.1007/978-1-4939-2438-7_1].
 B) The physical structure of the coronavirus virion, including the components determined by the conserved structural proteins S, E, M and N.
+This figure was adapted from “FullTemplateName”, by BioRender.com (2020), retrieved from https://app.biorender.com/biorender-templates.
```

Comment:
Obviously we should fill in the template name :)

Reply:
"Full template name" is "Human Coronavirus Structure"

Reply:
```suggestion
This figure was adapted from “Human Coronavirus Structure”, by BioRender.com (2020), retrieved from https://app.biorender.com/biorender-templates.
```